dist/APIManager.js

```diff
@@ -0,0 +1,77 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+
+    constructor(renderer)
+     {
+         this.cityData = []
+         this.renderer = renderer
+    }
+    getDataFromDB()
+    {
+            $.ajax({
```

Comment:
try to use axios in your next project. It is a lot cleaner and a lot more people use it